scripts/zones/Rolanberry_Fields/mobs/Ravenous_Crawler.lua

```diff
@@ -9,6 +9,10 @@ entity.onMobInitialize = function(mob)
     mob:addMod(xi.mod.REGAIN, 100)
```

Comment:
Have you gotten any capture data to suggest what value to put in there for that? That is the main reason I didn't adjust that.

Reply:
Doesn't even seem like Store TP. In the Ravenous Crawler video, the crawler never really lands a hit but it's frequently using Sticky Thread. It's also obviously not regain because I don't see any of the nov 2009 NMs using a TP move on pull like a mob with regain would do. Doesn't really seem like an obvious answer here to me.

Reply:
Sticky thread might be on a timer?

Reply:
Adjusted to store tp on Ravenous Crawler and Eldritch Edge.